src/components/sbb-calendar/sbb-calendar.scss

```diff
@@ -60,18 +64,40 @@
   @include sbb.text-s--regular;
 
   display: flex;
-  justify-content: center;
   align-items: center;
   margin: auto;
-  text-align: center;
+  height: var(--sbb-calendar-control-view-change-height);
   text-transform: capitalize;
   cursor: pointer;
+  padding-inline: var(--sbb-spacing-fixed-5x) var(--sbb-spacing-fixed-2x);
+  border-radius: var(--sbb-border-radius-infinity);
+  background-color: var(--sbb-calendar-control-view-change-background);
+  color: var(--sbb-calendar-control-view-change-color);
+  transition-duration: var(--sbb-calendar-cell-transition-duration);
+  transition-timing-function: var(--sbb-calendar-cell-transition-easing-function);
+  transition-property: background-color, padding-block-end;
+
+  &:disabled {
+    --sbb-calendar-control-view-change-background: var(--sbb-color-milk-default);
+    --sbb-calendar-control-view-change-color: var(--sbb-calendar-cell-disabled-color);
+
+    cursor: unset;
+  }
 
   &:focus-visible:not([data-focus-origin='mouse'], [data-focus-origin='touch']) {
     @include sbb.focus-outline;
 
     outline-offset: var(--sbb-spacing-fixed-1x);
-    border-radius: var(--sbb-border-radius-2x);
+  }
+
+  @include sbb.hover-mq {
+    &:not(:active, :disabled):hover {
+      padding-block-end: 0.0625rem;
```

Comment:
I remember we once had flickering problems when shifting content with padding when using screens with worse resolution than our mac books :). I can't test it now, but maybe we should test that too. The way to go was to use transform translation then.

Reply:
Should I apply the same change also on the day cell?

Reply:
if you would change one, then also the other please ;) can you additionally use pxtorem function?
But as written above, I did not test your scenario, maybe it's not even a problem, I was just remembering the other case (which was button and header action as I remember)